gef.py

```diff
@@ -3899,16 +3899,12 @@ def get_memory_alignment(in_bits: bool = False) -> int:
 def clear_screen(tty: str = "") -> None:
     """Clear the screen."""
     global gef
-    if not tty:
-        gdb.execute("shell clear -x")
-        return
 
-    # Since the tty can be closed at any time, a PermissionError exception can
-    # occur when `clear_screen` is called. We handle this scenario properly
     try:
-        with open(tty, "wt") as f:
-            f.write("\x1b[H\x1b[J")
+        sys.stdout.write("\x1b[H\x1b[J")
     except PermissionError:
```

Comment:
Does the `try/except` still makes sense since we don't open the tty anymore?

Reply:
Fixed by https://github.com/hugsy/gef/commit/730e41844aeb1185f97e98e3e6c739462a544e39